// [[Reason: Symbol merges with global eval and is not written to declarations.]] ////

//// [tests/cases/conformance/parser/ecmascript5/StrictMode/parserStrictMode8.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -1,15 +1,20 @@
 
 
 //// [parserStrictMode8.d.ts]
+declare function eval(): invalid;
 
 /// [Errors] ////
 
 parserStrictMode8.ts(2,10): error TS1100: Invalid use of 'eval' in strict mode.
+parserStrictMode8.ts(2,10): error TS9007: Function must have an explicit return type annotation with --isolatedDeclarations
 
 
-==== parserStrictMode8.ts (1 errors) ====
+==== parserStrictMode8.ts (2 errors) ====
     "use strict";
     function eval() {
              ~~~~
 !!! error TS1100: Invalid use of 'eval' in strict mode.
+             ~~~~
+!!! error TS9007: Function must have an explicit return type annotation with --isolatedDeclarations
+!!! related TS9031 parserStrictMode8.ts:2:10: Add a return type to the function declaration
     }
\ No newline at end of file
